### ISSUE TITLE
When notifications have long text, only one line is shown

### ISSUE THREAD

**Issue by valllllll2000** [akvo/akvo-flow-mobile#519]
We should show all the lines by using a different notification style:
`.setStyle(new Notification.BigTextStyle().bigText(longText)) `

**Comment by janagombitova** [comment]
Here is an old issue https://github.com/akvo/akvo-flow-mobile/issues/125 that is a duplicate. Let's continue with the implementation here.

**Comment by valllllll2000** [comment]
Applying the big text style only works stating android api 4.1, before that we have 2 options:
1) Make the text shorter
2) Use custom view to display the notification. Will try this and see how it actually looks.

**Comment by valllllll2000** [comment]
After multiple attempts, using custom views does not seem to work so I would suggest to make the error string shorter instead of
<string name="error_assignment_text">Assignment in Akvo FLOW dashboard required</string>
replaced by
<string name="error_assignment_text">Assignment in FLOW dashboard required</string>
The second version does not get cut off.

**Comment by janagombitova** [comment]
@valllllll2000 can the error say: "Assignment required" or is that too short?

**Comment by valllllll2000** [comment]
It could be fine but depends on the users, will they know what it refers to? Or maybe how about 
'User assignment required'
or
'Flow user assignment required'
or 
'Dashboard user assignment required'
we would have to replace dashboard by the new name

**Comment by janagombitova** [comment]
That is a good point. How about 'Flow assignment required' 

We do not link the assignments to users yet but to devices, and we will not have to switch the word dashboard for the "new" one.

**Comment by valllllll2000** [comment]
I like "Flow dashboard assignment required" even shorter!

**Comment by valllllll2000** [comment]
Test plan:
Download manually a monitored survey on you device via the settings.
Try syncing it.
You should see the error notification. It should be visible entirely or if not there should be a way to expand it. (On latest phones with android 7.0 press the little arrow inside the notification).

**Comment by valllllll2000** [comment]
The translations will need to be reviewed once this issue is merged

**Comment by janagombitova** [comment]
Looking at this particular notification case, the implementation passes the test plan. There are more cases where we need to tackle the length of the message, see https://github.com/akvo/akvo-flow-mobile/issues/526#issuecomment-266408577

**Comment by janagombitova** [comment]
@valllllll2000 all right. I have copied the image to this new issue https://github.com/akvo/akvo-flow-mobile/issues/560

This implementation is ready for release 👍
